azure-pipelines.yml

```diff
@@ -74,6 +75,7 @@ jobs:
     displayName: 'List installed dependencies'
   - script: |
       source activate test_env
+      export NREL_API_KEY=$(nrelApiKey)
```

Comment:
@wholmgren was the secret variable set as "nrelApiKey"? The linux/MacOS tests don't seem to be picking up the API key correctly and failing. I need to fix the Windows version "export" equivalent command as a separate issue.

Reply:
<img width="508" alt="Screen Shot 2020-01-30 at 9 35 31 AM" src="https://user-images.githubusercontent.com/4383303/73469631-e742b580-4343-11ea-9314-63a65c34ad52.png">

Reply:
I have a PR to my pvlib-python fork from the same branch that this PR is from [and Azure sees the variable.](https://dev.azure.com/ctstark/pvlib-test/_build/results?buildId=55&view=logs&j=06aebe0f-636a-5f93-19d5-0c37b58d55c8&t=0878738c-c430-5911-4cfd-99d3ab677436&l=20) You can see from [the latest Azure run that it's not able to see this variable.](https://dev.azure.com/solararbiter/pvlib%20python/_build/results?buildId=2533&view=logs&j=ee50eb0a-7467-5494-0f35-e3b863355bb0&t=2efe8f58-8c60-5d45-8189-b00aa6aac1e4&l=79) 

Can you double check that the variable is set in the pvlib-python pipeline? I'm guessing that you have access to multiple pipelines. Maybe an trailing whitespace?

Sorry to keep bugging you about this. If you'd rather wait until I have Azure Pipelines access I'm happy jumping into that.